### ISSUE TITLE
Loads of warning from compilation

### ISSUE THREAD

**Issue by arichiardi** [Malabarba/fancy-narrow#19]
Hello @Malabarba,
I am using your awesome tool together with `demo-it`, and, man..thanks! This is what I was looking for exactly.

I get warnings, a lot of warnings, when I try to compile it though. Not a big deal, but worth reporting it:

```
Warning (bytecomp): reference ot free variable `fancy-narrow--end`
Warning (bytecomp): reference ot free variable `fancy-narrow--beginning`
...
```

Thanks a lot again!

**Comment by Malabarba** [comment]
Thanks for the report.
I no longer support this package, though, so I probably won't be a ble to fix it. =/

**Comment by arichiardi** [comment]
Oh that's to bad, but good to know.

**Comment by dakra** [comment]
Think https://github.com/Malabarba/fancy-narrow/pull/20 was meant to say 'Fix #19' and this can be closed?!

**Comment by arichiardi** [comment]
Yeah that was the plan ...
